lebedigital/demonstrator_scripts/kpi_from_fem.py

```diff
@@ -0,0 +1,87 @@
+from lebedigital.unit_registry import ureg
+import numpy as np
+import pint_pandas
+import pint
+import pandas as pd
+
+
+def kpi_from_fem(df,limit_temp):
+    """
+    compute KPIs from simulation output
+
+    the maximum reached temperature and the time is extracted and the difference to the limit temperature given
+    the time when the yield value equals zero is approximated
```

Comment:
I think that needs further explanation, I guess you define a yield surface as the elastic limit based on the maximum return the maximum for any time step here, correct?

Reply:
correct. I added a more detailed description